examples/typescript/aws-s3-static-website/main.ts

```diff
@@ -0,0 +1,107 @@
+import { Construct } from "constructs";
+import { App, TerraformStack, TerraformVariable, TerraformAsset, TerraformOutput } from "cdktf";
+import { AwsProvider, s3 } from "./.gen/providers/aws";
+
+
+class MyStack extends TerraformStack {
+  constructor(scope: Construct, name: string) {
+    super(scope, name);
+
+    // Template Variables
+    const awsRegion = new TerraformVariable(this, 'awsRegion', {
```

Comment:
Please make sure you run a linter or otherwise verify that this example stays consistent with our code style; for example, our standard is double quotes for all strings, and there are indentation issues in a couple of places in this file.